### PR TITLE
Focus editor in replies and forwdared messagess

### LINKED ISSUE #2373: Reply issues
When you reply to an email, there are some small issues which are quite annoying when summed up:

- [x] The "To" field is focused, even though the recipient is already filled
- [x] You are prefilled in the "cc" field and need to manually remove yourself
- [ ] The text body has no space above and you need to insert newlines yourself. Expected would be focus on the first line, and 2 empty lines below until the original message.
- [ ] ~~If you are replying from the unified inbox, the wrong account is selected. This might be related to yourself being prefilled in cc~~ → separated out into https://github.com/nextcloud/mail/issues/2428

cc @nextcloud/mail I’d say this is some crucial polishing for 18.

### PR DESCRIPTION
**Pull request by GretaD**
Ref #2373

### CODE REVIEW

File: src/components/TextEditor.vue [hunk 1]
```diff
@@ -127,6 +131,9 @@ export default {
 					priority: 'highest',
 				}
 			)
+			if (this.focus) {
+				logger.debug('focusing TextEditor')
```
Comment (jancborchardt) [inline src/components/TextEditor.vue]: This should probably be removed? :)

**Comment by jancborchardt** [comment]
Hmm, I can’t really confirm that it fixes anything? Also I still have myself in cc when replying to mails.

**Comment by ChristophWurst** [comment]
> Also I still have myself in cc when replying to mails.

https://github.com/nextcloud/mail/pull/2454

**Comment by ChristophWurst** [comment]
> Hmm, I can’t really confirm that it fixes anything?

Have you built the app? What happens when you try to reply to a message? What is focused? Do you see the debug log about the focus?

File: src/components/TextEditor.vue (hunk 1 above)
Comment (ChristophWurst) [reply src/components/TextEditor.vue]: I'm fine with leaving those as they aid debugging of bug reports.

**Comment by GretaD** [comment]
@jancborchardt its working now, can you please recheck?

**Pull request merged**